### PR TITLE
draft-ietf-secevent-http-push last call feedback

### PR DESCRIPTION
**Pull request by mscurtescu**
Editorial changes based on feedback from these threads:
* https://mailarchive.ietf.org/arch/msg/id-event/mG-Uto9grX_La5lmi5ZR64X8EmI
* https://mailarchive.ietf.org/arch/msg/id-event/EfLHdOri6BiquWE4YKYQqedW9kA
* https://mailarchive.ietf.org/arch/msg/id-event/a7ikHnrCjl8mjdce3nJBuMUGvS8

### CODE REVIEW

File: draft-ietf-secevent-http-push.xml
Comment (selfissued) [inline draft-ietf-secevent-http-push.xml]: Add a comma after "document".

File: draft-ietf-secevent-http-push.xml
Comment (selfissued) [inline draft-ietf-secevent-http-push.xml]: Add "a SET" before "at its own discretion".  Currently the sentence doesn't say what it is that would be accepted or rejected.

**Review by selfissued — APPROVED** [review]
Please apply my minor suggestions before merging.

File: draft-ietf-secevent-http-push.xml
Comment (mscurtescu) [reply draft-ietf-secevent-http-push.xml]: done

File: draft-ietf-secevent-http-push.xml
Comment (mscurtescu) [reply draft-ietf-secevent-http-push.xml]: done

**Comment by mscurtescu** [comment]
Mike, please merge when you get a chance. I cannot merge since I don't have write access.

**Review by selfissued — APPROVED** [review]
Looks good

**Pull request merged**
